setupDB.py

```diff
@@ -0,0 +1,93 @@
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import Column, Integer, Float, Date, String, VARCHAR
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+import csv
+import pandas as pd
+
+# app = Flask(__name__)
+# app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:////tmp/test.db'
+# db = SQLAlchemy(app)
+
+Base = declarative_base()
+
+
+class Conferencedb(Base):
+    __tablename__ = 'womenInTechResources'
+    __table_args__ = {'sqlite_autoincrement': True}
```

Comment:
out of curiosity what is the settings for ? :)

Reply:
Line 19 was added to avoid sqlite reusing id's that might have been deleted. Does this case apply to our database? (can be removed if unnecessary)

Reply:
ahh I see okay - in that case we can just leave it in there :)